src/extensions/cellpack/model.ts

```diff
@@ -477,30 +506,57 @@ async function loadMembrane(plugin: PluginContext, name: string, state: State, p
         const url = Asset.getUrlAsset(plugin.managers.asset, `${params.baseUrl}/membranes/${name}.bcif`);
         b = b.apply(StateTransforms.Data.Download, { url, isBinary: true, label: name }, { state: { isGhost: true } });
     }
-
-    const membrane = await b.apply(StateTransforms.Data.ParseCif, undefined, { state: { isGhost: true } })
-        .apply(StateTransforms.Model.TrajectoryFromMmCif, undefined, { state: { isGhost: true } })
-        .apply(StateTransforms.Model.ModelFromTrajectory, undefined, { state: { isGhost: true } })
-        .apply(StructureFromAssemblies, undefined, { state: { isGhost: true } })
-        .commit({ revertOnError: true });
-
-    const membraneParams = {
-        representation: params.preset.representation,
+    const props = {
+        type: {
+            name: 'assembly' as const,
+            params: { id: '1' }
+        }
     };
-
-    await CellpackMembranePreset.apply(membrane, membraneParams, plugin);
+    if (params.source.name === 'id' && params.source.params !== 'MycoplasmaGenitalium.json'){
```

Comment:
Not too happy that there is a specific filename buried here. The info what kind of membrane this is should be in the packings data.

Reply:
It was temporary, the new push should resolve this issue.